english/covid-19/2/2-A.md

```diff
@@ -0,0 +1,5 @@
+You stayed in the room for 24 hours feared about the figures. Slowly, your eyes adjusted. After taking a long nap, you woke up as the sunlight shone through the chings of the door.
+
```

Comment:
> At least make one progressive link here which leads to somewhere else other than (../WIP.md)

same here (although it is good that you are helping others come up with the idea of how to progress but at least there should be one link which should go somewhere (maybe a small ending 🤷‍♂️) )

Reply:
Thanks for your comments. But, I don't know how to progress the story. That's why I opened this PR. So, let any others complete it 😉